client/signup/steps/user/index.jsx

```diff
@@ -265,6 +266,7 @@ export class UserStep extends Component {
 				suggestedUsername={ this.props.suggestedUsername }
 				handleSocialResponse={ this.handleSocialResponse }
 				isSocialSignupEnabled={ this.props.isSocialSignupEnabled }
+				isSocialFirst={ isSocialFirst }
```

Comment:
I've mentioned the fact that we don't pass any `oauth2_client_id` in the url of this `wpcc` flow in the patch. Depending on whether it's on purpose, we could simply rely on the presence of this parameter to show the Google button first.

Reply:
I'm not super comfortable hiding those kind of decisions into non-explicit rules. But I'm not 100% on the query param option either. I guess we could create a specific flow if we're not comfortable touching the wpcc one.

Reply:
It's on purpose as `oauth2_client_id` is used for analytics + customizing the UX only. We don't need either of those here.